did-utils/src/methods/did_peer/util.rs

```diff
@@ -84,13 +83,71 @@ fn abbrv_service(value: &mut Value, abbrv_key: &dyn Fn(&str) -> String, abbrv_va
     }
 }
 
+pub(super) fn validate_input_document(diddoc: &DIDDocument) -> Result<(), DIDPeerMethodError> {
+    // The document must not be empty
```

Comment:
Can you source this rule?

Reply:
It is obvious that creating a `did:peer:4` with an empty document is useless and must be avoid. Aren't you ok with that ?

Reply:
The wording makes it sound like it is normative.

Please can you update the comment from "// The document must not be empty" to "// The document **should** not be empty"?

Besides, even though it might seem useless, I personally don't think it must be avoided.